### ISSUE TITLE
help with errors

### ISSUE THREAD

**Issue by mikemanyoung** [ankane/eps#19]
I've followed the simple boiler plate for creating and training a model, and I keep getting this error message -- have some hints on what this means so I can debug?

[LightGBM] [Fatal] The number of features in data (59) is not the same as it was in training data (103).

I've verified that each item in my data has the same number of features, so not sure what this means. It seems that this message is coming from a linked lib call, so couldn't get into the code:

https://github.com/ankane/lightgbm/blob/master/lib/lightgbm/booster.rb#L139

**Comment by ankane** [comment]
Hey @mikemanyoung, can you share the issue and fix in case others run into this error?

**Comment by mikemanyoung** [comment]
@ankane I wasn't able to get this error again. Do you know what this error means, however?

**Comment by ankane** [comment]
Haven't seen that one before, but there's a chance the fix for #20 addresses it (there was an issue with how text features were computed for the validation set).
